src/Geocoder/Provider/Chain.php

```diff
@@ -38,6 +40,9 @@ public function geocode($address)
     {
         $exceptions = [];
         foreach ($this->providers as $provider) {
+            if($provider instanceof LocaleAwareProvider && $this->getLocale() !== null){
```

Comment:
Problem is when using GeocoderBundle. 
For exemple, I got a default locale for Bing Map
```
providers:
    bing_maps:
       api_key:  ~
       locale:    fr_FR
```

I'd like some times set a different locale, but if not, it should use the default one, not erase the default by null